src/login/systemd-user.m4

```diff
@@ -2,6 +2,8 @@
 #
 # Used by systemd --user instances.
 
+account required pam_unix.so
```

Comment:
If ew add pam_unix in for "account", shouldn't we also add it in for the others? 

(i don't know want to suggest we should, I am just wondering this myself)